### DISCUSSION TITLE
Show gameplay cursor when changing gameplay cursor size in settings

### DISCUSSION THREAD

**Discussion by ziomek64** [ppy/osu#17563] [Ideas] [answered]
As it says in the title. When you hold and adjust the size of gameplay cursor it would be nice if the arrow changed to your gameplay cursor and showed the actual size so you don't have to go the map and check it.

**Comment by peppy** [comment, marked as answer]
You can start autoplay and watch it change. Eventually we'll provide a preview for settings.
